src/Abp.AspNetCore/AspNetCore/Threading/HttpContextCancellationTokenProvider.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Abp.Dependency;
+using Abp.Threading;
+using Microsoft.AspNetCore.Http;
+using System.Threading;
+
+namespace Abp.AspNetCore.Threading
```

Comment:
Since `HttpContextCancellationTokenProvider` depends on `Microsoft.AspNetCore.Http`, let's move it to `Abp.AspNetCore.Http.Threading`?

We should move `HttpContextClientInfoProvider` and `HttpRequestEntityChangeSetReasonProvider` too, in a separate PR.

Reply:
> Since HttpContextCancellationTokenProvider depends on Microsoft.AspNetCore.Http, let's move it to Abp.AspNetCore.Http.Threading?
You mean a new namespace named Http under Abp.AspNetCore and Threading namespace under it. Then moving

- `HttpContextCancellationTokenProvider`
- `HttpContextClientInfoProvider`
- `HttpRequestEntityChangeSetReasonProvider`

under this namespace right?
I can do it with an other PR after this one.

Reply:
Only `HttpContextCancellationTokenProvider` will be under `Http.Threading`:
- `HttpContextCancellationTokenProvider`: `Threading` -> `Http.Threading`
- `HttpContextClientInfoProvider`: `Mvc.Auditing` -> `Http.Auditing`
- `HttpRequestEntityChangeSetReasonProvider`: `EntityHistory` -> `Http.EntityHistory`

@hikalkan Do you agree?

Reply:
@acjh I have moved those classes to places you suggested. I think the rest of the PR is fine, what do you think ?

Reply:
I think we still need https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5111#discussion_r376685328.

There are other classes that should be moved too — let me do this in a separate PR.

Reply:
Actually, there are many classes in `Abp.AspNetCore` that depend on `Microsoft.AspNetCore.Http` even though only some have `Http` in their class name.

Since it would be considerable a breaking change, shall we revert the move?